zipline/data/bundles/csvdir.py

```diff
@@ -175,10 +175,13 @@ def _pricing_iter(csvdir, symbols, metadata, divs_splits, show_progress):
         for sid, symbol in enumerate(it):
             logger.debug('%s: sid %s' % (symbol, sid))
 
+            fname = symbol + '.csv'
             try:
-                fname = [fname for fname in files
-                         if '%s.csv' % symbol in fname][0]
-            except IndexError:
+                dfr = read_csv(os.path.join(csvdir, fname),
```

Comment:
Should we remove the second call to `read_csv()` below on line 185? We're already calling it here and exiting early if we don't have the file we want.